tests/Aspire.Hosting.Azure.Tests/Snapshots/AzureStorageExtensionsTests.ResourceNamesBicepValid.verified.bicep

```diff
@@ -26,11 +26,20 @@ resource blobs 'Microsoft.Storage/storageAccounts/blobServices@2024-01-01' = {
   parent: storage
 }
 
+resource queues 'Microsoft.Storage/storageAccounts/queueServices@2024-01-01' = {
+  parent: storage
+}
```

Comment:
This is broken. The required property name is unset and there's no way to set it...

![image](https://github.com/user-attachments/assets/c1713fa9-a31e-488d-8798-c370178011c6)
![image](https://github.com/user-attachments/assets/a8438a66-b457-4018-9d21-990d3b92d05a)

Reply:
FYI - @tg-msft @m-nash

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/49672

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/49898 has been resolved, however, we're still waiting for a new version of https://www.nuget.org/packages/Azure.Provisioning.Storage to be published.